### ISSUE TITLE
I can't see my cables in the new version.

### ISSUE THREAD

**Issue by WillySeifert** [mattieserver/netbox-topology-views#515]
### NetBox version

v4.0.6

### Topology Views version

v4.0.0

### Steps to Reproduce

Hello Guys,
First of all: thank you for creating the update for netbox 4.0+. 
Now I was finally able to upgrade my whole netbox system. 

I ran into an issue tho.
When I create the topology map, I cant see my cables. I made sure that it is selcted in the filter. 
I can only see the logical connections. In the old version I was able to see both. The cables and the logical connections.

Is this a bug or did I just make a mistake? It is very much possible, that the mistake is on my end. In this case, you might have advice for me anyway.

![image](https://github.com/netbox-community/netbox-topology-views/assets/147381059/63ca6203-fc10-4697-9454-63d977e6a846)

![image](https://github.com/netbox-community/netbox-topology-views/assets/147381059/17308e04-580c-4634-899d-d59354f77e2d)

Thank you for your work! :)

### Expected Behavior

Cables and logical connections are displayed

### Observed Behavior

If I uncheck the logical connections, I don't see any devices, since unconnected devices are not supposed to be displayed. 
If I uncheck the logical connections AND check the unconnected devices, I can see my devices but not their cables.

**Comment by mattieserver** [comment]
I am not able to reproduce this.

Can you verify that the device role, site or other filters include all the needed devices?
For example if I use the filters from your screenshot and don't include the 'cabling' device role that I use for the patch panels I also don't see the cables.

You might also test with the 'show neighbors' options.

**Comment by WillySeifert** [comment]
I don't think that the device roles are the issue. As I mentioned, as soon as I toggle the logical connections on, the devices are displayed. The filter for the device role remained the same. 

Weirdly enough; it works now. 
I clicked through my cables and device roles, without changing anything, just searching and when I returned to the topology, the cables where there. I didn't change any of the filters. 

So my issues is solved, I guess. Unfortunately I can't really explain why it works now.
Thank you for your answer tho :)
